website/docs/guides/docs/docs-create-doc.mdx

```diff
@@ -174,6 +176,29 @@ It is possible to use:
 
 :::
 
+#### Relationship between `id` and `slug` and document path
```

Comment:
Way too verbose. Just say that you can provide explicit docs URLs using the `slug` front matter, which is inferred from the doc id and file path.


To me, we don't need another heading, this should be part of `#### Customizing Document URLs`